site/src/pages/[shortname]/blocks/[block-slug].page.tsx

```diff
@@ -106,6 +107,7 @@ type BlockPageProps = {
   schema: BlockSchema;
   sliderItems: BlockMetadata[];
   exampleGraph: BlockExampleGraph | null; // todo fix typing
+  isUnsupportedBlock: boolean;
```

Comment:
Could we just call `isNonV0_2Block` inside `block-data-container`? we're passing this `isUnsupportedBlock` prop through this page but aren't using it

Reply:
Agree. Done in [755c63a](https://github.com/blockprotocol/blockprotocol/pull/417/commits/755c63a237bdd58cb3764c31ac7d3293decc4ef7)